doc/aws-deployment.md

```diff
@@ -0,0 +1,386 @@
+3.  Click the "Add" button
+4.  Under "Select the property to add", select "feature.use_open_id_connect"
+5.  Under "Value", select "Yes".
+6.  Click "Add property" to save.
+
+## User groups
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Manage groups"
+3.  Click the "Add" button.
+4.  Create a new group called "readers", then click "Add group"
+5.  Repeat for a new group called "editors".
+
+## User accounts
+
+Once groups are added, create the user accounts.
+
+1.  Sign in as the administrator
+2.  Click on the user menu (top-right corner) and select "Manage users"
+3.  Click "Add"
+4.  Enter the details for each user (see below), including the user group they
+    belong in.
+5.  Click "Add user" to save.
+6.  Repeat for each user.
+
+**Note**: at this point in the setup, you may not have SMTP setup yet so the
+new users will not automatically get their initial password directly. It is
+possible to workaround this though:
+
+1.  Go to the "Emails" page
+2.  Each email should have a status of "Failed", but the body of each
+    registration email will be listed.
+3.  Click the "Show more" link in each row to view the initial password for
+    each user account, then send it to them manually and securely.
+4.  As each user signs in, they will be forced to change their password to
+    something of their choice.
+
+The following user accounts should be created:
+
+Name                Email                       Environments            Groups
+------------------- --------------------------- ----------------------- ---------------
+Peter Monks         peter.monks@systemc.com     TEST, TRAINING, LIVE    Administrators
+Mike Hewlett        mike.hewlett@systemc.com    TEST, TRAINING, LIVE    Administrators
+Oliver Butler       oliver.butler@systemc.com   TEST, TRAINING, LIVE    Administrators
+Angela Faulding     angela.faulding@nhs.net     TEST, TRAINING, LIVE    Administrators
+Cath Chilcott       cath.chilcott@nhs.net       TEST                    Editors
+Kate Palmer-Lilley  kate.palmer-lilley@nhs.net  TEST, TRAINING, LIVE    Administrators
+Steve Bowring       steve.bowring@nhs.net       TEST                    Readers
+
+## Folders and Permissions
+
+The user groups are used to determine the permissions they are allowed to do on
+folders in Mauro – this is the primary permissions mechanism used.
+
+1.  Go to the main model view page, the "Browse" link
+2.  Add a new root folder (not version controlled) called "Data Dictionary"
+3.  Select the "Data Dictionary" folder in the tree so the details appear on
+    the right hand side
+4.  Click the "User & Group Access" toolbar button then in the dialog, set the
+    groups and their roles.
+
+
+*   Someone in the "Readers" group only has read access to this folder and all
+    models below
+*   Someone in the "Editors" group has read and write access to this folder and
+    all models below
+*   Administrators have full access regardless
```

Comment:
It might be useful to include a screenshot of Mauro UI showing the user access roles of the folder once completed.

Reply:
Added screenshot from the TRAINING deployment